src/khoj/processor/conversation/openai/gpt.py

```diff
@@ -32,6 +33,8 @@ def extract_questions(
     def _valid_question(question: str):
         return not is_none_or_empty(question) and question != "[]"
 
+    location = f"{location_data.city}, {location_data.state}, {location_data.country}" if location_data else "Unknown"
```

Comment:
Should this be set to empty string if `location_data` is not available like what's been done for offline chat? The user may provide their location via a chat message or notes. Having "location = unknown" in the prompt may confuse (some) chat models